bin/startup.sh

```diff
@@ -1,5 +1,45 @@
 #!/bin/bash
 
+# Setting Mellon key, cert and metadata files
+if grep -q "libapache2-mod-auth-mellon" "${HOME}/Aptfile" ; then
+  
+  export APACHE_DIR="${APACHE_DIR:-$HOME/vendor/apache2}"
+  export MELLON_DIR="${MELLON_DIR:-$APACHE_DIR/mellon}"
+  mkdir -p "${MELLON_DIR}"
+
+  if [[ ( ! -n "${MELLON_SP_METADATA}" ) ]]; then
```

Comment:
```suggestion
  if [[ -z "${MELLON_SP_METADATA}" ]]; then
```
To check if a variable is empty, use this notation (and same for below.

Was the new version tested with a real app?

Reply:
Fixed.
Yes, it was tested and confirmed it works